rdtools/normalization.py

```diff
@@ -34,6 +34,8 @@ def pvwatts_dc_power(poa_global, P_ref, T_cell=None,
     gamma_pdc: numeric, default is None
         Linear array efficiency temperature coefficient [1 / degree celsius].
 
+    Note: All series are assumed to be right-labeled
```

Comment:
Mike - code looks good and I was able to run the example .ipny document.  My only question here is on units  - pvwatts_dc_power uses kW as default power input, and normalize_with_pvwatts uses Wh as the unit for energy.  I guess I would expect that if you're passing Wh in as energy, you should specify system power in W.  Or pass in energy in kWh and system size in kW.  If you keep things as-is, I would specify units explicitly in the docstring.